### ISSUE TITLE
Problems when trying to Maven-pull (Eclipse)

### ISSUE THREAD

**Issue by ehx-v1** [bjornharrtell/jts2geojson#10]
This is a followup issue to #9, mainly a question but correlated with a documentation request. Upon trying to establish the Maven pull, I noticed that I forgot to add the repository. I added it, but it's still failing to pull. Could you please review my POM for what else I'm missing out, and write an explanation for how to pull with Maven Eclipse so other users won't run in the same problems?

Here's my POM:

```
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.ptvgroup.haeg.neo4j</groupId>
  <artifactId>Spatial</artifactId>
  <version>0.0.1-SNAPSHOT</version>
  <repositories>
    <repository>
      <id>neo4j-contrib-releases</id>
      <url>https://raw.github.com/neo4j-contrib/m2/master/releases</url>
      <releases>
        <enabled>true</enabled>
      </releases>
      <snapshots>
        <enabled>false</enabled>
      </snapshots>
    </repository>
    <repository>
      <id>neo4j-contrib-snapshots</id>
      <url>https://raw.github.con/neo4j-contrib/m2/master/snapshots</url>
      <releases>
        <enabled>false</enabled>
      </releases>
      <snapshots>
        <enabled>true</enabled>
      </snapshots>
    </repository>
    <repository>
      <id>ossrh</id>
	  <url>https://oss.sonatype.org/content/repositories/snapshots</url>
	  <releases>
        <enabled>false</enabled>
      </releases>
      <snapshots>
        <enabled>true</enabled>
      </snapshots>
    </repository>
  </repositories>
  <dependencies>
   <dependency>
    <groupId>org.neo4j</groupId>
    <artifactId>neo4j</artifactId>
    <version>2.3.0</version>
   </dependency>
   <dependency>
     <groupId>org.glassfish.external</groupId>
     <artifactId>apache-commons</artifactId>
     <version>3.0-b68</version>
   </dependency> 
   <dependency>
     <groupId>org.neo4j</groupId>
     <artifactId>neo4j-spatial</artifactId>
     <version>0.15-neo4j-2.3.0</version>
   </dependency>
   <dependency>
   	  <groupId>org.wololo</groupId>
	  <artifactId>jts2geojson</artifactId>
	  <version>0.8.0</version>
	</dependency>
  </dependencies>
</project>
```

**Comment by ehx-v1** [comment]
When I pulled version 0.7.0, it all of a sudden worked. However, it should be documented that one must pull 0.7.0, and not 0.8.0.

**Comment by ehx-v1** [comment]
I think I open a new issue for the documentation. There's a bit of stuff here that's off topic for the documentation request.

**Comment by bjornharrtell** [comment]
There is no released 0.8.0. The latest released version is 0.7.0.

**Comment by ehx-v1** [comment]
That's what I figured out. But it's not necessarily obvious that the version in the Master branch POM is not yet released. Thus my documentation request.
